client/payment-details/dispute-details/dispute-due-by-date.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { dateI18n } from '@wordpress/date';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import classNames from 'classnames';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import type { EvidenceDetails } from 'wcpay/types/disputes';
+
+const DisputeDueByDate: React.FC< {
+	dueBy: EvidenceDetails[ 'due_by' ];
```

Comment:
Can we reduce coupling by making this type `number`?

Reply:
Great idea – resolved in 8388a9bdde0597ce154c22d07678af98e781d737.